apps/src/sharedComponents/footer/CopyrightDialog.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import AccessibleDialog from '@cdo/apps/sharedComponents/AccessibleDialog';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import i18n from '@cdo/locale';
+
+import styles from '@cdo/apps/sharedComponents/footer/copyright-dialog.module.scss';
+
+export interface CopyrightDialogProps {
+  isOpen: boolean;
+  closeModal: () => void;
+}
+
+const CopyrightDialog: React.FC<CopyrightDialogProps> = ({
```

Comment:
Do you think we can add some unit tests for this component as well?

Reply:
Since the Dialog is a pretty static component, that uses AccessibleDialog, I didn't consider adding more test coverage than the storybook tests.